modules/lx-autoscale/watchmaker-lx-autoscale.template.cfn.yaml

```diff
@@ -1049,13 +1043,13 @@ Resources:
               export AWS_DEFAULT_REGION=${AWS::Region}
 
               # Get pip
-              PYPI_URL=${PypiIndexUrl}
-              curl --silent --show-error --retry 5 -L ${CfnGetPipUrl} | python3 - --index-url="$PYPI_URL"
+              python3 -m ensurepip --upgrade --default-pip
```

Comment:
so according to the docs, `esnsurepip` commands operate without an internet connection, but is that still the case when used with the  `--upgrade` flag? Assuming it's referencing the `PYPI_URL` env var should that be set prior to calling this command?

Reply:
Yes, it upgrades pip only to the version included in the installed version of python. Without the --upgrade flag, if pip were already installed but at an earlier version, it would just noop. With the --upgrade flag, it updates pip to the included/packaged version.

Reply:
👍 sounds good